### PR TITLE
Update README for more clear instructions

### PR DESCRIPTION
**Pull request by PotatoPeeler3000**
Add more details on how to setup the logger from source, like that you will need Git, Java, and Gradle. Added the commands to install them to make things easier for the user

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,45 +6,73 @@ IHMC Robot Data Logger
...
+---
+
+## Setting up the Logger from source (Ubuntu 22.04 / Magewell)
 
-## Ubuntu 22.04 (recommended)
 - Install Ubuntu Desktop 22.04 LTS (it's convenient to have a GUI to work with)
-	- Make sure to install OpenSSH
+    - Make sure to install OpenSSH
+- Install Git:
+    - `sudo apt install git`
```
Comment (PotatoPeeler3000) [inline README.md:26]: Added these setup instructions for people who set up the logger from source

File: README.md
Comment (PotatoPeeler3000) [inline README.md:1]: The `Setting up the Logger from source` is the only change in the file. The rest was formatting

**Comment by ds58** [comment]
Can you undo all formatting changes to the README so it only contains the actual content changes?

File: README.md [hunk 2]
```diff
@@ -6,45 +6,73 @@ IHMC Robot Data Logger
...
+    - `sudo apt update`
+    - `sudo apt install openjdk-17-jdk`
+- Install Gradle: (we will install curl to help with this)
+    - `sudo apt install curl`
+    - `cd ~/Downloads`
+    - `curl -sL https://services.gradle.org/distributions/gradle-8.1.1-all.zip -o gradle.zip\`
+    - `unzip -q gradle.zip -d gradle`
+    - `sudo mv gradle/gradle-8.1.1/ /opt/`
+    - `sudo ln -s /opt/gradle-8.1.1/ /opt/gradle`
+    - `sudo ln -s /opt/gradle/bin/gradle /usr/bin/gradle`
```
Comment (calvertdw) [inline README.md:37]: You should make these code blocks

**Comment by PotatoPeeler3000** [comment]
> Can you undo all formatting changes to the README so it only contains the actual content changes?

@ds58 This was a result of running the IHMC formatter. Now if you ran it again you wouldn't get any changes which is ideal.

File: README.md [hunk 3]
```diff
@@ -6,45 +6,73 @@ IHMC Robot Data Logger
...
 
 Depending on how the logger is deployed, it can be setup to automatically start when the computer boots, or restart at the beginning of each day.
-To start the logger navigate to `/opt/ihmc/logger/bin/` and run `./IHMCLogger`. This will start the logger manually if you have it configured to not run on boot or restart at the beginning of each day.
+To start the logger navigate to `/opt/ihmc/logger/bin/` and run `./IHMCLogger`. This will start the logger manually if you have it configured to not run on boot
+or restart at the beginning of each day.
 The Logger can also be run as a service, to see if thats the case this command `ps aux | grep java` tells you if java processes are running.
 The service should be `ihmc-robot-data-logger.service`.
 
-Before starting the logger, you need to setup the camera file, and the host file. These are important to capturing video feeds, as well as logging from specific hosts.
-The steps below walk through how to set those two files up. They should be setup on the remote computer that is doing the logging. This is why we recommend using Ubuntu Desktop because setting these up are much simpler if you have access to the GUI.
```
Comment (calvertdw) [inline README.md:47]: Not a fan of theses changes that add line breaks. I think if you want line breaks, put one sentence per line.

**Review by ihmc-rosie — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -90,7 +121,17 @@ hosts:
...
+Alternatively, you can start `SCSVisualizer` from `ihmc-robot-data-visualizer` and add hosts using the GUI. After you close the visualizer, the hosts you added
+will be saved  `~/.ihmc/ControllerHosts.yaml`. You can copy that file to the logger if it is on a different computer.
+
+---
+
+## Viewing Logs in SCS2
 
-Alternatively, you can start `SCSVisualizer` from `ihmc-robot-data-visualizer` and add hosts using the GUI. After you close the visualizer, the hosts you added will be saved  `~/.ihmc/ControllerHosts.yaml`. You can copy that file to the logger if it is on a different computer.
- 
+Simulation Construction Set 2 (SCS2) is used to view logs. This repository lives
```
Comment (PotatoPeeler3000) [inline README.md:133]: Added a section that tries to explain how logs can be viewed in SCS2

**Pull request merged**